<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewStoryDialog</class>
 <widget class="QDialog" name="NewStoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Story Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>57</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Summary</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="summaryEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="minimumSize">
         <size>
          <width>58</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>58</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="reporterEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="minimumSize">
         <size>
          <width>57</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>57</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="7" colspan="2">
       <widget class="QTextEdit" name="descriptionEdit"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Estimation</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QComboBox" name="estComboBox">
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QComboBox" name="impComboBox">
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Component</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QComboBox" name="comComboBox">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_6">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QComboBox" name="typComboBox">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Sprint</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QComboBox" name="sprComboBox">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QComboBox" name="verComboBox">
        <property name="minimumSize">
         <size>
          <width>141</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QTextEdit" name="htdEdit"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>How To Demo</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Importance</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewStoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewStoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
